roles/virt-install/tasks/create_vm.yml

```diff
@@ -8,6 +8,7 @@
 - name: 'Gather the list of existing VMs'
   command: virsh list --name
   register: existing_vms
+  check_mode: false
```

Comment:
Why the need for `check_mode` here?

Reply:
this makes sure that the command runs also despite a potential `--check` option and that the command just after doesn't fail because the registered variable is empty. More would need to be done to get a complete run working in check mode but it goes already a longer way with this option.